com.unity.multiplayer.mlapi/Tests/Editor/InterestTests.cs

```diff
@@ -0,0 +1,344 @@
+using System.Collections.Generic;
```

Comment:
It doesn't have to be in this PR, but I'd be interested to see end to end tests using the MultiInstanceHelper, see from a user's perspective what would be needed to setup these.

Reply:
Actually, I could look into it for perf tests